extensions/wikia/InsightsV2/InsightsHooks.class.php

```diff
@@ -16,7 +16,7 @@ public static function init() {
 	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
 		global $wgEnableGlobalShortcutsExt;
 
-		if ( !empty( $wgEnableGlobalShortcutsExt ) ) {
+		if ( !empty( $wgEnableGlobalShortcutsExt ) && \Wikia\GlobalShortcuts\Helper::shouldDisplayGlobalShortcuts() ) {
```

Comment:
It's better to use `use Wikia\GlobalShortcuts\Helper` declaration in the beginning of a class definition.